api/zulip/__init__.py

```diff
@@ -38,15 +38,13 @@
 from six.moves import urllib
 import logging
 import six
+from typing import Any, Dict
 
 
 __version__ = "0.2.5"
 
 logger = logging.getLogger(__name__)
 
-# Check that we have a recent enough version
-# Older versions don't provide the 'json' attribute on responses.
-assert(LooseVersion(requests.__version__) >= LooseVersion('0.12.1'))
```

Comment:
Hmm.  This code is relevant on older platforms like Ubuntu Precise, which hasn't reached EOL yet.  Can you just do a `#type: ignore # LooseVersion not supported` instead?